integrations/nifi/nifi-nar-bundles/nifi-core-bundle/nifi-core-processors/src/main/java/com/thinkbiganalytics/util/TableType.java

```diff
@@ -80,10 +80,13 @@ public String deriveLocationSpecification(Path tableLocation, String source, Str
         Validate.notNull(entity, "entity expected");
 
         Path path = tableLocation.resolve(source).resolve(entity).resolve(tableSuffix);
+        String location = path.toString().replace(":/","://");
```

Comment:
If a path contains // slashes, Paths APIs take //  fixed the path by removing one slash. But in URIs like `s3://bucket-name` we do require double forward slashes. Paths objets handles many other scenarios but lack a forward slash. After path is calculated, extra forward slash is added to fix the URI path.